packages/vuepress-theme-odyssey/layouts/Layout.vue

```diff
@@ -25,16 +39,23 @@ import { Fragment } from "vue-fragment";
 import { resolveNav } from "../utils";
 import "../styles/index.scss";
 
+const BANNER_ID = "ods-beta-banner";
+
 export default {
   components: {
     Fragment,
+    DocsBanner: () => import("../components/DocsBanner.vue"),
     DocsSidebar: () => import("../components/DocsSidebar.vue"),
+    DocsLink: () => import("../components/DocsLink.vue"),
     DocsTemplateHome: () => import("../templates/DocsTemplateHome.vue"),
     DocsTemplateIndex: () => import("../templates/DocsTemplateIndex.vue"),
     DocsTemplatePlain: () => import("../templates/DocsTemplatePlain.vue"),
     DocsTemplateComponent: () =>
       import("../templates/DocsTemplateComponent.vue")
   },
+  data: () => ({
+    showBanner: true
```

Comment:
Notice once you dismiss the banner, you'll have a slight visual of the banner without content when refreshing the page. One way to avoid this is to default to `false` here, then show the banner when needed (instead of by default).

Reply:
Woah, this is a great catch. Thanks for calling it out. Will fix.

Reply:
Fixed.